.github/workflows/tests.yml

```diff
@@ -72,6 +72,10 @@ jobs:
         run: |
           yarn lint
 
+      - name: ESLint (ignore failures for now)
+        run: |
+          yarn eslint || echo "ESLint still failing... Fine for now!"
+
```

Comment:
Making this an always passing step in the CI for now, and plan on tackling issues as I spot them in files that we touch over the next few months so we can switch over fully away from TSLint.